src/main/java/com/alkemy/ong/exception/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.alkemy.ong.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.ZonedDateTime;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+
+	@ExceptionHandler(value={ResourceNotFoundException.class})
+	public ResponseEntity<ErrorResponse> handleResourceNotFoundException(ResourceNotFoundException exception){
+		ErrorResponse error= new ErrorResponse();
+		error.setStatus(HttpStatus.NOT_FOUND.value());
+		error.setMessage(exception.getMessage());
+		error.setTimeStamp(ZonedDateTime.now());
+		return ResponseEntity.status(HttpStatus.NOT_FOUND).body(error);
+	}
+
+	@ExceptionHandler
```

Comment:
Maybe is not a good idea to handle every single Exception as BAD REQUEST. It could be better to respond with "unknown internal error" and respond with status 500.